pkg/agent/controller/networkpolicy/allocator_test.go

```diff
@@ -184,15 +184,15 @@ func TestWorker(t *testing.T) {
 		expectedErr       error
 	}{
 		{
-			"delete-rule-with-async-delete-interval",
+			"delete-rule-with-min-async-delete-interval",
 			nil,
-			5 * time.Millisecond,
+			10 * time.Millisecond,
 			rule,
 			1,
 			nil,
 		},
 		{
-			"delete-rule-with-flow-poll-interval",
+			"delete-rule-with-async-delete-interval",
 			nil,
 			1 * time.Millisecond,
```

Comment:
I think you should rename `testDeleteInterval` to `testMinDeleteInterval` and then add a comment for each test:

* `delete-rule-with-min-async-delete-interval`: 10ms is larger than `testMinDeleteInterval`, rule should take 10ms to be deleted
* `delete-rule-with-async-delete-interval`: 1ms is smaller than `testMinDeleteInterval`, rule should take `testMinDeleteInterval` (5ms) to be deleted

Reply:
Done. Thanks for pointing the inconsistency with the test global variables. Modified the code to make it consistent with `allocator.go`.